Typeracer/src/components/GameData.js

```diff
@@ -22,6 +54,57 @@ function GameData() {
     
     const wordsPerMinute = gameData.CalculativeStatistics.WordsPerMinute;
     const accuracy = gameData.CalculativeStatistics.Accuracy;
+
+    function generateUUID() {
+        // generating a random playerID
+        return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
```

Comment:
Hmm, not sure this is the best approach. Not only is it pretty complicated, in C# a unique user id can be created with a simple Guid.NewGuid() method. why complicate things and make the guid generation inside the front-end logic?

Reply:
What I mean is that the responsibility for creating a player and all data attributed to that player should likely be handled in the back-end

Reply:
Agreed. This was used primarily because I had problems when deserializing .json file. PlayerID generation should be implemented in the server.